onnxruntime/test/onnx/main.cc

```diff
@@ -682,7 +682,67 @@ select from 'TF8', 'TF16', 'UINT8', 'FLOAT', 'ITENSOR'. \n)");
-        ORT_TSTR("sce_none_weights_expanded")};
+        ORT_TSTR("sce_none_weights_expanded"),
+        // float 8 types
+        ORT_TSTR("cast_FLOAT_to_BFLOAT16"),
+        ORT_TSTR("cast_BFLOAT16_to_FLOAT"),
+        ORT_TSTR("castlike_FLOAT_to_BFLOAT16"),
+        ORT_TSTR("castlike_BFLOAT16_to_FLOAT"),
+        ORT_TSTR("castlike_FLOAT_to_BFLOAT16_expanded"),
+        ORT_TSTR("castlike_BFLOAT16_to_FLOAT_expanded"),
+        ORT_TSTR("castlike_FLOAT_to_STRING"),
+        ORT_TSTR("castlike_FLOAT_to_STRING_expanded"),
+        ORT_TSTR("cast_FLOAT8E5M2_to_FLOAT"),
+        ORT_TSTR("cast_FLOAT8E5M2_to_FLOAT16"),
+        ORT_TSTR("cast_FLOAT8E5M2FNUZ_to_FLOAT"),
+        ORT_TSTR("cast_FLOAT8E5M2FNUZ_to_FLOAT16"),
+        ORT_TSTR("cast_FLOAT_to_FLOAT8E4M3FN"),
+        ORT_TSTR("cast_FLOAT_to_FLOAT8E4M3FNUZ"),
+        ORT_TSTR("cast_FLOAT_to_FLOAT8E5M2"),
+        ORT_TSTR("cast_FLOAT_to_FLOAT8E5M2FNUZ"),
+        ORT_TSTR("cast_no_saturate_FLOAT16_to_FLOAT8E4M3FN"),
+        ORT_TSTR("cast_no_saturate_FLOAT16_to_FLOAT8E4M3FNUZ"),
+        ORT_TSTR("cast_no_saturate_FLOAT16_to_FLOAT8E5M2"),
+        ORT_TSTR("cast_no_saturate_FLOAT16_to_FLOAT8E5M2FNUZ"),
+        ORT_TSTR("cast_no_saturate_FLOAT_to_FLOAT8E4M3FN"),
+        ORT_TSTR("cast_no_saturate_FLOAT_to_FLOAT8E4M3FNUZ"),
+        ORT_TSTR("cast_no_saturate_FLOAT_to_FLOAT8E5M2"),
+        ORT_TSTR("cast_no_saturate_FLOAT_to_FLOAT8E5M2FNUZ"),
+        ORT_TSTR("castlike_FLOAT8E4M3FNUZ_to_FLOAT"),
+        ORT_TSTR("castlike_FLOAT8E4M3FNUZ_to_FLOAT_expanded"),
+        ORT_TSTR("castlike_FLOAT8E4M3FN_to_FLOAT"),
+        ORT_TSTR("castlike_FLOAT8E4M3FN_to_FLOAT_expanded"),
+        ORT_TSTR("castlike_FLOAT8E5M2FNUZ_to_FLOAT"),
+        ORT_TSTR("castlike_FLOAT8E5M2FNUZ_to_FLOAT_expanded"),
+        ORT_TSTR("castlike_FLOAT8E5M2_to_FLOAT"),
+        ORT_TSTR("castlike_FLOAT8E5M2_to_FLOAT_expanded"),
+        ORT_TSTR("castlike_FLOAT_to_FLOAT8E4M3FN"),
+        ORT_TSTR("castlike_FLOAT_to_FLOAT8E4M3FNUZ"),
+        ORT_TSTR("castlike_FLOAT_to_FLOAT8E4M3FNUZ_expanded"),
+        ORT_TSTR("castlike_FLOAT_to_FLOAT8E4M3FN_expanded"),
+        ORT_TSTR("castlike_FLOAT_to_FLOAT8E5M2"),
+        ORT_TSTR("castlike_FLOAT_to_FLOAT8E5M2FNUZ"),
+        ORT_TSTR("castlike_FLOAT_to_FLOAT8E5M2FNUZ_expanded"),
+        ORT_TSTR("castlike_FLOAT_to_FLOAT8E5M2_expanded"),
+        ORT_TSTR("castlike_FLOAT_to_STRING_expanded"),
+        ORT_TSTR("castlike_FLOAT_to_STRING"),
+        ORT_TSTR("castlike_FLOAT_to_BFLOAT16"),
+        ORT_TSTR("castlike_BFLOAT16_to_FLOAT_expanded"),
+        ORT_TSTR("castlike_BFLOAT16_to_FLOAT"),
+        ORT_TSTR("cast_FLOAT16_to_FLOAT8E4M3FN"),
+        ORT_TSTR("cast_FLOAT16_to_FLOAT8E4M3FNUZ"),
+        ORT_TSTR("cast_FLOAT16_to_FLOAT8E5M2"),
+        ORT_TSTR("cast_FLOAT16_to_FLOAT8E5M2FNUZ"),
+        ORT_TSTR("cast_FLOAT8E4M3FNUZ_to_FLOAT"),
+        ORT_TSTR("cast_FLOAT8E4M3FN_to_FLOAT"),
+        ORT_TSTR("cast_FLOAT8E5M2FNUZ_to_FLOAT"),
```

Comment:
this shows up twice in this list. perhaps sorting them alphabetically would make it easier to spot duplicates

Reply:
I'll do that.